docker/build_scripts/build.sh

```diff
@@ -51,42 +51,27 @@ esac
 # Decided not to clean at this point: https://github.com/pypa/manylinux/pull/129
 yum -y update
 
-# EPEL support (for cmake28 & yasm)
-yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+# Software collection (for devtoolset-8)
```

Comment:
As 'centos-release-scl' is removed on AArch64 then maybe move that 'yum install' to same line as installing 'epel-release-latest-7'?

Reply:
mmh, my comment is probably unclear in the uninstall phase. You need to install & then erase for `yum install devtoolset-8-...` to work. This is strange but what I observed.

Reply:
"centos-release-scl-rh" stays on aarch64 - maybe this is package you need to install. Will take a look at it tomorrow.